### ISSUE TITLE
PDF viewer: garbled Chinese script

### ISSUE THREAD

**Issue by adrik29** [SpeciesFileGroup/taxonworks#2220]
The built in PDF viewer is choking with Chinese characters.
This is my original:
![image](https://user-images.githubusercontent.com/46284943/116837999-1ed18a80-aba3-11eb-8e1a-f7682f470463.png)

This is what I get with viewer [maybe a configuration of my browser????].
![image](https://user-images.githubusercontent.com/46284943/116838012-2db83d00-aba3-11eb-8b24-0ed56f372a29.png)

**Comment by mjy** [comment]
There is nothing we can do with the built in viewer really, it's not our library/code., and many (many) things won't work with it.

**Comment by debpaul** [comment]
@adrik29 for these then, you have to use your PDF viewer, yes? I'm going to close this, but I do wonder if we need some sort of note (somewhere?) that let's people know the viewer doesn't work for all character sets?

**Comment by mjy** [comment]
Indeed, we need a big ovious section in help that indicates that the in-browser PDF viewer is just a library we use, and it's not optimum.
